kpi/views.py

```diff
@@ -520,7 +525,33 @@ def preview(self, request, *args, **kwargs):
             return Response(response_data, template_name='preview_error.html')
 
 
-class AssetViewSet(viewsets.ModelViewSet):
+class AssetVersionViewSet(NestedViewSetMixin, viewsets.ModelViewSet):
+    model = AssetVersion
+    lookup_field = 'uid'
+    filter_backends = (
+            AssetOwnerFilterBackend,
```

Comment:
Yeah. true. by the time I got to this I just wanted to be done with it so i could get back to formpack  reports. But I made a new issue and @esmail has already started to use `RelatedAssetPermissionsFilter` for this in #1164.  I just didn't want it to be a blocker to quick debugging of users' reports